### PR TITLE
[gh-pages][ZEPPELIN-2303] Fix lately published package ordering

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### What is this PR for?
I added pagination below Helium package list in https://zeppelin.apache.org/helium_packages.html by #2126. But after this, "Lately published" page doesn't show actual latest package first. It's because while i'm paginating the packages, I should've made ordering by `published` date first before [slicing package array](https://github.com/apache/zeppelin/compare/gh-pages...AhyoungRyu:ZEPPELIN-2303/fixLatelyPublishedOrdering?expand=1#diff-c2c3af7bcb6c1faa231eaf25c15d3732R63). This PR will fix it. 

### What type of PR is it?
Bug Fix | gh-pages

### What is the Jira issue?
[ZEPPELIN-2303](https://issues.apache.org/jira/browse/ZEPPELIN-2303)

### How should this be tested?
Run the website(`gh-pages`) locally as described in [here](https://github.com/apache/zeppelin/tree/gh-pages#run-website) and compare the ordering like below. 

```
1. "zeppelin-sigma-spell" : "2017-03-10T09:31:56.802Z"
2. "zeppelin-d3-spell" : "2017-03-08T10:08:51.022Z"
3. "zeppelin-markdown-spell" : "2017-03-07T19:17:35.975Z"
4. "zeppelin-echo-spell" : "2017-03-07T11:37:44.283Z"
5. "zeppelin-csv-spell" : "2017-02-28T04:57:05.463Z"
6. "zeppelin-json-spell" : "2017-02-28T04:49:27.897Z"
7. "zeppelin-highcharts-heatmap" : "2017-02-14T12:46:02.732Z"
...

Last. "zeppelin-bubblechart" : "2017-01-23T20:42:34.373Z"
```

tl;dr **"zeppelin-sigma-spell"** should be the first & **"zeppelin-bubblechart"** should be the last 

### Screenshots (if appropriate)
 - Before 
before

 - After 
after

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by sravan-s** [comment]
Hi, I guess calling it 'recently published' instead of 'lately published' is a better slang

**Comment by AhyoungRyu** [comment]
@sravan-s Thanks for the suggestion. Addressed in 34dfec8a41a0e998fcc8c030860a6c796d11df86 :)
screen shot 2017-03-23 at 6 33 06 pm

**Comment by AhyoungRyu** [comment]
CI failure is not related with this change since it's based on `gh-pages` branch. 
Merge into `gh-pages` if there are no more comments.
